rules/common.js

```diff
@@ -0,0 +1,42 @@
+module.exports = {
+  env: {
+    browser: true,
+    es6: true,
+    mocha: true,
+    jasmine: true,
+    node: true,
+    commonjs: true,
+  },
+  parserOptions: {
+    ecmaVersion: 2018,
+    sourceType: 'module',
+  },
+  rules: {
+    // -- Error
+    'no-shadow': 2,
+    'default-case': 2,
+
+    // -- Warning
+    'class-methods-use-this': 1,
+    'no-else-return': 1,
+    'no-param-reassign': 1,
+    'no-underscore-dangle': 1,
+    'no-unused-vars': 1,
+    semi: [1, 'always'],
+    'import/no-unresolved': 1,
+    'import/prefer-default-export': 1,
+    'prefer-destructuring': 1,
+    'lines-between-class-members': 1,
```

Comment:
Is `1` "warning" or `true`? I see `2` above. And then I see something like `skipComments: true`.

Reply:
`0` == `off`, `1` == `warning`, `2` == `error`